components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/ServerApplicationManagementService.java

```diff
@@ -2028,4 +2043,12 @@ private boolean isApplicationBasedOutboundProvisioningEnabled() {
         }
         return applicationBasedOutboundProvisioningEnabled;
     }
+
+    private void blockRenameAppsToSystemReservedApps(String newAppName, String oldAppName) {
+
+        if ((CONSOLE_APP.equals(newAppName) || MY_ACCOUNT_APP.equals(newAppName)) &&
+                !StringUtils.equals(newAppName, oldAppName)) {
```

Comment:
Is this to enable the updates for Console and MyAccount?

Reply:
Yes, if a normal app is trying to rename to console or myaccount.